README.md

```diff
@@ -35,9 +36,75 @@ npm install -g @unstoppabledomains/resolution
 
 It should install binary named resolution in the default folder for your package manager. You can check it by running command `resolution -V` in command line. If everything is fine you will see the version installed.
 
-# Usage
+## Usage
+
+Create a new project.
+
+```shell
+mkdir test-out-resolution && cd $_
+yarn init -y
+yarn add @unstoppabledomains/resolution
+```
+
+Make a file, `script.js`.
+
+```javascript
+const {default: Resolution} = require('@unstoppabledomains/resolution')
+const resolution = new Resolution()
+function resolve(domain, currency) {
+  resolution.address(domain, currency)
+    .then(address => console.log(domain, 'resolves to', address))
+    .catch(console.error)
+}
+resolve('resolver.eth', "ETH")
+resolve('brad.zil', "ZIL")
+```
+
+Execute the script.
+
+```
+node script.js
+# brad.zil  resolves to zil1yu5u4hegy9v3xgluweg4en54zm8f8auwxu0xxj
+# resolver.eth resolves to 0xD3ddcCDD3b25A8a7423B5bEe360a42146eb4Baf3
+```
+
+### How to resolve
+
+Resolution library provides a way to resolve a domain name using a direct blockchain call.
+For this purpose there are two main methods to look for: ** Resolution.resolve ** and ** Resolution.address **
+
+#### Resolution#resolve
```

Comment:
@mvlabat  Should we make the description of the methods a part of the code via jsdoc?

Reply:
Yeah, I don't like the idea of documenting methods in the README too, but I've copy-pasted it as-is from our old doc, as we agreed not to change a lot at this point

Reply:
If you think that it's totally superfluous, I'm okay with just dropping this section. We'll need to make a more comprehensive guide about using our library later anyway

Reply:
@mvlabat let's left this on Readme for now and create a task for javadoc coverage. Any documentation is better than no docs at all :)

Reply:
We have a jsdoc generation that can be accessed from here: https://unstoppabledomains.github.io/resolution/
There is also a reference to the same webpage on this readme. 
I vote for dropping this section off and replace it with something relating to the dweb resolution. Getting an ipfsHash can be a good example or maybe a section that shows how to get the gundb username/public key for p2p chat system?

Reply:
@JohnnyJumper agreed, this would be nice to have. Won't do it as a part of this PR, but I noted it for our future docs improvements